test/format/en_US.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE formattest SYSTEM "../formattest.xsd">
+<formattest locale="en-US">
+    <test>
+        <params>
+            <param name="relationship" type="String"><value>brother</value></param>
+        </params>
+        <source>I didn't find <var name="relationship" inflect="indefinite"/> in your contacts. What is your <var name="relationship" inflect="genitive"/> name?</source>
+        <print>I didn't find a brother in your contacts. What is your brotherʼs name?</print>
+    </test>
+    <test>
+        <params>
+            <param name="relationship" type="String"><value>aunt</value></param>
+        </params>
+        <source>I didn't find <var name="relationship" inflect="indefinite"/> in your contacts. What is your <var name="relationship" inflect="genitive"/> name?</source>
+        <print>I didn't find an aunt in your contacts. What is your auntʼs name?</print>
+    </test>
+    <test>
+        <params>
+            <param name="object" type="String"><value>light</value></param>
```

Comment:
How does the translation of the word `light` get into this message?

Reply:
This is user vocabulary.  I could have easily used the phrase "light on the front porch" or "Stanisław's party light". If you're dealing with something like HomeKit, the naming of various objects is up to the user.

It may not be translated. If it's translated, it would require additional infrastructure. It's stored in a SemanticFeatureModel, but I didn't define how it goes into a SemanticFeatureModel in these test cases. Fluent seems to have a similar concept to the translated phrases, but I think it's put into the same file of messages.

Reply:
I added Russian tests to highlight how a SemanticConcept would work as a quantity. I used Latin text to highlight that it's using the declared data instead of a lexical dictionary, how the number changes pronunciation and how the grammatical case changes as the value changes.

If I wanted to be complete with the testing in Russian, I'd vary the grammatical gender and test more of the speak lines.

I picked Russian because it's much more complicated when compared to many other languages.